internal/command.go

```diff
@@ -62,10 +62,11 @@ import (
 
 func NewConfluentCommand(cfg *config.Config) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     pversion.CLIName,
-		Short:   fmt.Sprintf("%s.", pversion.FullCLIName),
-		Long:    getLongDescription(cfg),
-		Version: cfg.Version.Version,
+		Use:           pversion.CLIName,
+		Short:         fmt.Sprintf("%s.", pversion.FullCLIName),
+		Long:          getLongDescription(cfg),
+		Version:       cfg.Version.Version,
+		SilenceErrors: true,
```

Comment:
No, this is because the by default, errors are printed by cobra in black and white. I was hoping I could disable this and print my own (colored) errors manually. Unfortunately it's proving to be more difficult than I thought...